apps/site/test/site_web/plugs/glx_now_open_test.exs

```diff
@@ -22,24 +22,26 @@ defmodule SiteWeb.Plugs.GlxNowOpenTest do
   end
 
   describe "call/2" do
-    test "assigns glx_now_open? with false if current date is before opening date", %{conn: conn} do
+    test "assigns glx_stations_open with nil if current date is before opening date", %{
+      conn: conn
+    } do
       updated_conn = call(conn, now_fn: &now_before_fn/0)
 
-      assert updated_conn.assigns.glx_now_open? == false
+      assert updated_conn.assigns.glx_stations_open == nil
     end
 
-    test "assigns glx_now_open? with true if current date is between opening date and 3 months post opening",
+    test "assigns glx_stations_open with array of stations if current date is between opening date and 3 months post opening",
          %{conn: conn} do
       updated_conn = call(conn, now_fn: &now_between_fn/0)
 
-      assert updated_conn.assigns.glx_now_open? == true
+      assert updated_conn.assigns.glx_stations_open == "place-lech,place-unsqu,place-spmnl"
```

Comment:
We'll have to update this when the Medford branch's station open later this year. Might be sufficient to just ensure a truthy value with `assert updated_conn.assigns.glx_stations_open`